docs/run-taco.md

```diff
@@ -1,8 +1,19 @@
 ---
-title: Run Your Packaged Connector (TACO file)
+title: Run Your Connector
 ---
 
-Starting in 2019.4 Beta 1, you can load packaged connectors (otherwise known as TACO files). To use a connector in earlier Tableau versions, see  [Run Your "Under Development" Connector]({{ site.baseurl }}/docs/share)
+# Load order and class name collisions
+
+If a connector has the same class as a connector that has already been registered, the new connector will be rejected. This means that connectors loaded first have precedent when two connectors share the same class name.
+
+Tableau loads connectors by directory in the following order:
+1. Built-in Tableau connectors
+1. Connectors located in "My Tableau Repository/Connectors"
+1. (Optional) Connectors in the dev path specified by `-DConnectPluginsPath`
+
+# Run Your Packaged Connector (TACO file)
+
+Starting in 2019.4 Beta 1, you can load packaged connectors (otherwise known as TACO files). To use a connector in earlier Tableau versions, or to run an unpackaged connector, see [Run Your "Under Development" Connector]({{ site.baseurl }}/docs/run-taco#run-your-under-development-connector) below.
```

Comment:
```suggestion
Starting in 2019.4, you can load packaged connectors (otherwise known as TACO files). To use a connector in earlier Tableau versions, or to run an unpackaged connector, see [Run Your "Under Development" Connector]({{ site.baseurl }}/docs/run-taco#run-your-under-development-connector) below.
```